### ISSUE TITLE
Matrix(double[][] A) constructor

### ISSUE THREAD

**Issue by haietza** [JaysGitLab/cs5666-spr17-javatrix-c2m2#1]
Construct a matrix from a 2-D array.

**Comment by haietza** [comment]
@KaraBeason Please remember to delete the branch once it is merged into master. With 50 issues/branches, we're going to have a lot going on in this repo! :) Thank you!
